### DISCUSSION TITLE
Welcome to Chill Viking

### DISCUSSION THREAD

**Discussion by peterjokumsen** [chill-viking/.github#3] [Announcements]
Woot woot! `chill-viking` has more than a single member! 🥳

Welcome to the party, please feel free to play around! Feel free to fail as often as needed to find the solution you are trying to achieve.

- Do you have an idea for a project? 🤘 **[Start a discussion](https://github.com/orgs/chill-viking/discussions/categories/ideas)**, let's take that idea to its absolute limit and enjoy the adventure
- Do you think something in the organization's repos is stupid/misaligned? 🤣 _That's awesome!_ **[start a discussion](https://github.com/orgs/chill-viking/discussions/categories/ideas)**, let's find a way that isn't stupid/misaligned 😉
- Have you found something interesting? ❤️ *Sweeet!*, [start a discussion](https://github.com/orgs/chill-viking/discussions/categories/show-and-tell) so that we can be informed and share in the fun 🎉🎊🪅
- Have you got a question? 🤔🙋‍♂️🙋‍♀️ [start a discussion](https://github.com/orgs/chill-viking/discussions/categories/q-a) and hopefully it can be answered

This should be a space for open and honest sharing.

Here's hoping everybody has an incredible adventure in this space 🤘🎶🎇

**Comment by ProfPanga** [comment]
Peter's progressive thinking is concerning,
but I'm all in 🤷‍♂️
